### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by beatrizserrano**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,8 @@
 
 ## Galaxy Europe
 
-- Use this anywhere we're really talking about the European Galaxy project as a collective.
+- Use this anywhere we're really talking about the European Galaxy project as a collective, e.g. posters next to your ELIXIR/de.NBI logos.
```
Comment (bgruening) [inline README.md]: I would not mention this. Most people do not know what de.NBI is and maybe even ELIXIR outside of Europe.

Comment (beatrizserrano) [inline README.md]:
```suggestion
- Use this anywhere you refer to the European Galaxy project/community.
```

File: README.md [hunk 2]
```diff
@@ -7,9 +7,8 @@
...
 - Do not use this when referring to specific sub-teams fo the Galaxy Project Europe.
```
Comment (beatrizserrano) [inline README.md]:
```suggestion
- Do not use this when referring to specific sub-teams of the Galaxy Project Europe.
```

File: README.md [hunk 3]
```diff
@@ -23,9 +22,8 @@ Light Background                           | Dark Background
 
 ## Freiburg Galaxy
 
-- Use this anywhere we're really talking about the Freiburg Galaxy Team
-- Do not use this when referring to the collective team would be better
-- E.g. any articles/trainings produced by Freiburg Galaxy Team
+- Use this anywhere we're really talking about the Freiburg Galaxy Team, e.g. any articles/trainings produced by Freiburg Galaxy Team.
```
Comment (beatrizserrano) [inline README.md]:
```suggestion
- Use this anywhere you refer to the Freiburg Galaxy Team, e.g. any articles/training activities produced by the Freiburg Galaxy Team.
```

File: README.md [hunk 4]
```diff
@@ -23,9 +22,8 @@ Light Background                           | Dark Background
...
+- Do not use this when referring to the collective team.
```
Comment (beatrizserrano) [inline README.md]:
```suggestion
- Do not use this when referring to a broader community.
```

File: README.md [hunk 5]
```diff
@@ -39,10 +37,8 @@ Light Background                                       | Dark Background
 
 ## useGalaxy.eu
 
-- Use this anywhere where you want to direct users to our server, e.g.
-  trainings where you want to remind users "here is the URL" and stickers you
-  place which should be able to point users to our server with no further
-  text.
+- Use this anywhere where you want to direct users to our server, e.g. trainings where you want to remind users "here is the URL" and stickers you
```
Comment (beatrizserrano) [inline README.md]:
```suggestion
- Use this anywhere where you want to direct users to the European Galaxy server, e.g. trainings where you want to remind users "here is the URL" and stickers you
```

**Review by bgruening — APPROVED** [review]

**Pull request merged**